dnf-behave-tests/createrepo_c/duplicates.feature

```diff
@@ -0,0 +1,88 @@
+Feature: Tests createrepo_c with duplicate (same NEVRA) packages present
+
+
+Background:
+Given I create directory "/temp-repo/"
+  And I create directory "/temp-repo/subdir"
+  And I copy file "{context.scenario.repos_location}/createrepo_c-ci-packages/x86_64/package-0.2.1-1.fc29.x86_64.rpm" to "/temp-repo"
+
+
+Scenario: report all duplicate packages
+Given I create directory "/temp-repo/subdir2"
+  And I create directory "/temp-repo/subdir3"
+  And I copy file "{context.scenario.repos_location}/createrepo_c-ci-packages/x86_64/package-0.2.1-1.fc29.x86_64.rpm" to "/temp-repo/subdir"
+  And I copy file "{context.scenario.repos_location}/createrepo_c-ci-packages/x86_64/package-0.2.1-1.fc29.x86_64.rpm" to "/temp-repo/subdir2"
+  And I copy file "{context.scenario.repos_location}/createrepo_c-ci-packages/x86_64/package-0.2.1-1.fc29.x86_64.rpm" to "/temp-repo/subdir3"
+ When I execute createrepo_c with args "." in "/temp-repo"
+ Then the exit code is 2
+  And stdout is
+      """
+      Directory walk started
+      Directory walk done - 4 packages
+      Temporary output repo path: ./.repodata/
+      Preparing sqlite DBs
+      Pool started (with 5 workers)
+      Pool finished with errors
+      """
+  And stderr contains only lines
+  """
+  Warning: Package 'package-0:0.2.1-1.fc29.x86_64' has duplicate metadata entries, only one should exist
+  Warning:     Sourced from location: 'subdir2/package-0.2.1-1.fc29.x86_64.rpm'
+  Warning:     Sourced from location: 'subdir/package-0.2.1-1.fc29.x86_64.rpm'
+  Warning:     Sourced from location: 'package-0.2.1-1.fc29.x86_64.rpm'
+  Warning:     Sourced from location: 'subdir3/package-0.2.1-1.fc29.x86_64.rpm'
```

Comment:
The exact order of these items might be non-deterministic, I think it would depend on which files various threads in the threadpool manage to insert first.  We could sort the entries before printing them if that's a problem.

Reply:
Yes it was non-deterministic, I was adding a step for that `stderr contains only lines` which doesn't check the order of lines but with your update we don't need that. That is better. :slightly_smiling_face: 
Thank you.